fluentd/run.sh

```diff
@@ -123,13 +112,58 @@ if [ "${USE_MUX_CLIENT:-}" = "true" ] ; then
     DIV=`expr $DIV \* 2`
 fi
 
+# Default buffer_type: file if FILE_BUFFER_PATH is specified.
```

Comment:
Then should you have some code like 
```
BUFFER_TYPE=${FILE_BUFFER_PATH:+file}
```
which means "set BUFFER_TYPE to file if FILE_BUFFER_PATH is set, and if FILE_BUFFER_PATH is not set, BUFFER_TYPE will be empty"